maven-plugin/src/main/java/org/openmrs/maven/plugins/Build.java

```diff
@@ -56,10 +65,28 @@
      */
     private boolean buildOwa;
 
+    /**
+     * @parameter expression="${npmVersion}"
+     */
+    private String npmVersion;
+
+    /**
+     * @parameter expression="${nodeVersion}"
+     */
+    private String nodeVersion;
+
     private final static String FRONTEND_BUILDER_GROUP_ID = "com.github.eirslett";
     private final static String FRONTEND_BUILDER_ARTIFACT_ID = "frontend-maven-plugin";
     private final static String FRONTEND_BUILDER_VERSION = "1.0";
 
+    private final static String MAVEN_EXEC_PLUGIN_GROUP_ID = "org.codehaus.mojo";
+    private final static String MAVEN_EXEC_PLUGIN_ARTIFACT_ID = "exec-maven-plugin";
+    private final static String MAVEN_EXEC_PLUGIN_VERSION = "1.5.0";
+
+    public static final String PACKAGE_JSON_FILENAME = "package.json";
+    public static final String NODE_VERSION_KEY = "node-version";
+    public static final String NPM_VERSION_KEY = "npm-version";
```

Comment:
Using camelcase instead of '-' seems to be a convention in package.json see https://docs.npmjs.com/files/package.json